package.json

```diff
@@ -64,7 +66,13 @@
     "toposort": "^2.0.2",
     "ts-jest": "^24.0.2",
     "ts-node": "^8.3.0",
-    "typescript": "^3.5.3"
+    "typescript": "^3.5.3",
+    "prettier": "^1.18.2",
+    "eslint": "^6.4.0"
+  },
+  "peerDependencies": {
+    "eslint": "^6.0.0",
+    "prettier": "^1.18.2"
   },
```

Comment:
I'm not sure why `prettier` and `eslint` need to be included in both `devDependencies` and `peerDependencies`. Because these are `dependencies` of `@stencila/dev-config` it usually isn't necessary. So I wonder if this is a local setup thing.